File: src/main/java/client/database/data/CharacterRankData.java
package client.database.data;

public class CharacterRankData {
   private long lastLogin;

   private int loggedIn;

   private int move;

   private int rank;

   private int characterId;

   public CharacterRankData(long lastLogin, int loggedIn, int move, int rank, int characterId) {
      this.lastLogin = lastLogin;
      this.loggedIn = loggedIn;
      this.move = move;
      this.rank = rank;
      this.characterId = characterId;
   }

   public long getLastLogin() {
      return lastLogin;
   }

   public int getLoggedIn() {
      return loggedIn;
   }

   public int getMove() {
      return move;
   }

   public int getRank() {
      return rank;
   }

   public int getCharacterId() {
      return characterId;
   }
}

File: engine/model/src/main/scala/client/database/data/CharacterRankData.scala
package client.database.data

import java.sql.Timestamp
import java.util.Date

class CharacterRankData(private var _lastLogin: Date, private var _loggedIn: Int, private var _move: Int,
                        private var _rank: Int, private var _characterId: Int) {
  def lastLogin: Timestamp = _lastLogin.asInstanceOf[Timestamp]

  def loggedIn: Int = _loggedIn

  def move: Int = _move

  def rank: Int = _rank

  def characterId: Int = _characterId
}
